HW2_Islamova/calculator.py

```diff
@@ -0,0 +1,73 @@
+# calculator.py
+
+AVAILABLE_OPERATORS = ['+', '-', '*', '/']
+
+def add(a, b):
+    result = a + b
+    return result
+
```

Comment:
Можно было бы и
```suggestion
    return a + b
```
Не обязательно и не всегда нужно, особенно если вычисление длинное - но тут оно короткое и очевидное. Помогает читаемости кода.